Makefile

```diff
@@ -6,6 +6,6 @@ include .bootstrap/root/Makefile
 
 ## <<Stencil::Block(targets)>>
 post-stencil::
-	# Make sure that the shared orb is updated appropriately
-	@SKIP_VALIDATE=true make test
+	@# Make sure that the shared orb is updated appropriately
+	@SKIP_VALIDATE=true make test; exit 0
```

Comment:
This is the only manual change. Doesn't print out the comment, and allows `make post-stencil` to complete successfully if template snapshots need to be updated.